code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -17,11 +17,11 @@
 	. = -1		//hunters are sanic
 	. += ..()	//but they still need to slow down on stun
 
-/mob/living/carbon/alien/humanoid/hunter/handle_hud_icons_health()
-	..() //-Yvarov
-
+/mob/living/carbon/alien/humanoid/hunter/update_health_hud()
```

Comment:
Note to self; look into just fixing all of these up to be based on `living` or `carbon`'s `update_health_hud` thing; maybe based on percentages.